orm/auth.md

```diff
@@ -113,6 +113,55 @@ return [
 ];
 ```
 
+## Password hashing
+Password hashing must be handled by your application; Laravel's authentication
+and LaravelDoctrine will treat passwords as nothing more than strings. We would
+recommend decoupling the operation of hashing of the password (and any other
+procedures, like validating strength) from its storage by implementing a separate
+service to handle any password-related actions.
+
+```
+use \Illuminate\Contracts\Hashing\Hasher;
+
+class PasswordService
+{
+  private $hasher;
+  private $passwordStrengthValidator;
+
+  /**
+   * @param Hasher $hasher
+   * @param MyPasswordStrengthValidator $passwordStrength
+   */
+  public function __construct(
+    Hasher $hasher,
+    MyPasswordStrengthValidator $passwordStrength
+  ) {
+    $this->hasher = $hasher;
+    $this->passwordStrengthValidator = $passwordStrength
+  }
+
+  /**
+   * Validate and change the given users password
+   *
+   * @param User $user
+   * @param string $password
+   * @throws PasswordTooWeakException
+   * @return void
+   */
+  public function changePassword(User $user, $password)
+  {
+    if($this->passwordStrengthValidator->isStrongEnough($password))
```

Comment:
Could you make this method (changePassword) PSR-2 (code style) compatible. Rest is looking good 👍